pkg/netpol/eval/peer.go

```diff
@@ -14,4 +16,7 @@ type Peer interface {
 	String() string
 	// Kind returns a string of the peer kind in case the peer is a pod/workload, else it returns an empty string
 	Kind() string
+	// GetAllowedConnectionsToPod returns ConnectionSet of the allowed connections to the Pod object of the peer
+	// if it is a pod, else returns empty ConnectionSet
+	GetAllowedConnectionsToPod() k8s.ConnectionSet
```

Comment:
can this exposed function return `eval.Connection` instead of `k8s.ConnectionSet` ?

Reply:
since we can't import `eval`  from its internal package `k8s`.
I reverted adding the func of computing the allowed conns to a peer from the `eval.Peer` interface
and added a new `pe.AllowedConnectionsToPeer(peer)` which computes the conectionset of a pod and returns the connection object to the ingress_analyzer